addons/website/models/website.py

```diff
@@ -813,12 +814,15 @@ def _handle_views_and_pages(views):
             views = views - page_views
             if page_views:
                 dependencies.setdefault(page_model_name, [])
-                dependencies[page_model_name] += [{
+                for page in page_views.page_ids:
+                    new_dependency = {
                     'field_name': 'Content',
                     'record_name': page.name,
                     'link': page.url,
                     'model_name': page_model_name,
-                } for page in page_views.page_ids]
+                    }
+                    if new_dependency not in dependencies[page_model_name]:
```

Comment:
To avoid this error:

![owl_error](https://user-images.githubusercontent.com/114914971/209633258-64ecc373-8a47-4061-8131-c033099e5d78.png)